### PR TITLE
Styles init

### PR DESCRIPTION
**Pull request by alanshaw**
Initialise the styling work!

* Removes bootstrap
* Adds Tachyons
* Adds postcss imports and variables for build time variable replacement
* Adds palette variables
* Adds favicon
* Adds header and toggle buttons:

screen shot 2017-10-17 at 10 20 04

### CODE REVIEW

**Comment by olizilla** [comment]
👓

File: src/colors.css [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+  --blue-bayox: #50617B;
+  --bright-turquoise: #23e0f7;
+  --caribbean-green: #02caad;
+  --cloud-burst: #233855;
+  --dove-gray: #656464;
+  --firefly: #0e1d32;
+  --big-stone: #1a2c45;
+  --black-pearl: #081527;
+  --pigeon-post: #b5c5df;
+  --razzmatazz: #FB0D5B;
```
Comment (olizilla) [inline src/colors.css]: 👌 ✨

**Comment by olizilla** [comment]
In Chrome `Version 61.0.3163.100 (Official Build) (64-bit)`

screenshot 2017-10-17 11 04 23

I get the same behaviour on master.

screenshot 2017-10-17 11 06 41

**Comment by diasdavid** [comment]
@olizilla I believe you might be having a node boot problem. Clear all application storage on that localhost:3000, you probably have a lying old IPFS repo that is breaking IPFS init (check the console).

**Comment by olizilla** [comment]
Yep, deleted all the indexdbs and localstorages and it works!

screenshot 2017-10-17 11 18 25

**Pull request merged**